be/src/storage/lake/tablet_manager.cpp

```diff
@@ -311,6 +335,10 @@ StatusOr<TabletMetadataPtr> TabletManager::load_tablet_metadata(const string& me
     return reader.get_meta();
 }
 
+TabletMetadataPtr TabletManager::get_latest_tablet_metadata(int64_t tablet_id) {
```

Comment:
IMHO, this function name is kind of misleading, how about `get_latest_cached_tablet_metadata`

Reply:
Done